File: example/python/methods/filter_on_list.py
from pyspark.sql import DataFrame


def filter_on_list(df,targetCol,values):
	

    # Get the Scala API
    api = df._sc._jvm.com.example.api.FilterOnListAPI

    return DataFrame(api.filterOnList(df._jdf,targetCol,values), df.sql_ctx)

File: example/src/main/scala/com/example/api/filter_on_list.scala
package com.example.api

import java.util
import scala.collection.JavaConversions._
import org.apache.spark.sql.{DataFrame, Dataset, Row}
import com.example.FilterOnList

object FilterOnListAPI {
  

  def filterOnList(df: DataFrame,targetCol: String,values: List[Int]): DataFrame = {
    FilterOnList.filterOnList(df,targetCol,values)
  }
}